src/App.js

```diff
@@ -1,16 +1,42 @@
 import React from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import { useState } from 'react';
+
 
 const App = () => {
+
+  const initialCopy = chatMessages.map((bike) => {
+    return { ...bike };
+  });
+  
+  const [chatMessagesData, setChatMessagesData] = useState(initialCopy);
+  const [countLikes, setCountLikes] = useState(0);
+  
+  const updateLikes = (messageToUpdate)=>{
+    const messages = chatMessagesData.map((message)=>{
+    if (message.id === messageToUpdate.id){
+      return messageToUpdate;
+    }
+    return message;
+  });
```

Comment:
Make sure you're keeping a consistent indentation level in your code. Lines 23 - 30 should all be indented a level. Good code style helps with readability and maintainability. :)